databus-connector/src/main/java/com/sequenceiq/cloudbreak/sigmadbus/processor/AbstractDatabusRecordProcessor.java

```diff
@@ -63,8 +68,8 @@ public void init() {
             LOGGER.debug("Sigma DataBus endpoint is not set or stream processing is not enabled for {}", getDatabusStreamConfiguration().getDbusServiceName());
         } else {
             LOGGER.debug("Starting sigma databus worker for databus service: {} ", getDatabusStreamConfiguration().getDbusServiceName());
-            RoundRobinDatabusProcessingQueues<C> processingQueues = new RoundRobinDatabusProcessingQueues<C>(
```

Comment:
here i think the proper refactor is to delete RoundRobinDatabusProcessingQueues itself, and also hide that logic where we are creating the threads (same has done in streaming-common)